rfd/0119/README.md

```diff
@@ -79,23 +54,23 @@ to include entries for each attached subnet. (Using static routes for each
 destination subnet interacts more nicely with existing instances that have a
 primary NIC on the internet and are using its gateway as their default route.)
 
-There are two classes of fabric networks that can be attached to a network, and
-affect the routes that get created for it. There are fabrics that are located on
-the same Virtual Layer 2 (VL2) network (same datacenter, same VNET identifier,
-and same VLAN identifier), and there are fabrics that are located on a separate
-VL2, possibly in another datacenter.
+There are two classes of fabric networks that can be attached to a network,
+and affect the routes that get created for it. There are fabrics that are
+located on the same Virtual Layer 2 (VL2) network (same datacenter, same VNET
+identifier, and same VLAN identifier), and there are fabrics that are located
+on a separate VL2, possibly in another datacenter.
 
 The first class of networks can be made to work today with some manual
 configuration on the part of the customer. For these networks, we will return
-`"linklocal"` in the `"routes"` object to indicate that they can be treated like
-they are on the same virtual segment. (See [TRITON-265] and [TRITON-266] for
-supporting interface routes in Triton.)
+`"linklocal"` in the `"routes"` object to indicate that they can be treated
+like they are on the same virtual segment. (See [TRITON-265] and [TRITON-266]
+for supporting interface routes in Triton.)
 
-When attaching to the second class of networks, NAPI will select an IP address
-from the source network, save it in the `"overlay_router"` field, and use it as
-the next-hop address in the `"routes"` object. This IP address will be mapped in
-Portolan to a special MAC address recognized by the overlay devices. (See the
-Overlay Changes section for more here.)
+When attaching to the second class of networks, NAPI will select an IP
+address from the source network, save it in the `"overlay_router"` field, and
+use it as the next-hop address in the `"routes"` object. This IP address will
+be mapped in Portolan to a special MAC address recognized by the overlay
```

Comment:
I know we've been using 'special', but maybe 'designated' might be better? -- just to convey there's really nothing magical about it except in the target cache, it has a flag set to indicate VL3 processing is required for traffic to that MAC.

Reply:
Fixed s/special/designated/g.